lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
Also note that this is being used in production by the event service and the updates are working